vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java

```diff
@@ -25,4 +29,32 @@ static void fireClientEvent(Spreadsheet spreadsheet, String eventName,
                 true, eventName, JsonUtil.parse(jsonDataArray)));
     }
 
+    /**
+     * Ceates a Spreadsheet component with the given Excel file as the data
+     * source.
+     *
+     * @param fileName
+     *            the name of the file. The file must be in the test_sheets
+     *            folder.
+     * @return the test sheet file
+     */
+    static Spreadsheet createSpreadsheet(String fileName) {
+        File testSheetFile = getTestSheetFile(fileName);
+        try {
+            return new Spreadsheet(testSheetFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Could not create Spreadsheet", e);
+        }
+    }
+
+    static File getTestSheetFile(String name) {
```

Comment:
Should we add a javadoc for this method?

Reply:
Done